core/shared/lib/showdown/extensions/ghostimagepreview.js

```diff
@@ -1,5 +1,4 @@
 /* jshint node:true, browser:true */
```

Comment:
I can't remember why we have two copies of this file now. Did `ember-cli`'s `app.import(...)` fail with files that were outside the scope of the project? That sounds right..